environment.conf.json

```diff
@@ -112,6 +113,7 @@
   "QUESTIONS_ENDPOINT": "https://acc.api.data.amsterdam.nl/signals/v1/public/questions/",
   "ROLES_ENDPOINT": "https://acc.api.data.amsterdam.nl/signals/v1/private/roles/",
   "SEARCH_ENDPOINT": "https://acc.api.data.amsterdam.nl/signals/v1/private/search",
+  "SOURCES_ENDPOINT": "https://acc.api.data.amsterdam.nl/signals/v1/private/sources/",
```

Comment:
This prop should be conditionally required; when `fetchSourcesFromBackend` is true, this prop should be present. If `fetchSourcesFromBackend` is false, there is no need to provide a string of uri format.